broker-cli/broker-daemon-client/index.js

```diff
@@ -73,8 +74,12 @@ class BrokerDaemonClient {
       if (!this.username) throw new Error('No username is specified for authentication')
       if (!this.password) throw new Error('No password is specified for authentication')
 
-      // Go back to the ./broker-cli/certs directory from the current directory
-      this.cert = readFileSync(path.join(PROJECT_ROOT, this.certPath))
+      let certPathParts = this.certPath.split(path.sep)
```

Comment:
would add a comment JIC to let the dev know that we are trying to resolve the tilde